src/shared/components/annotation/genomeNexus/PolyPhen2.tsx

```diff
@@ -15,6 +15,8 @@ export function hideArrow(tooltipEl: any) {
 }
 
 export default class PolyPhen2 extends React.Component<IPolyPhen2Props, {}> {
+    static POLYPHEN2_URL:string = "http://genetics.bwh.harvard.edu/pph2/";
```

Comment:
does it support https?

Reply:
No. We are not pulling data from there though, just showing a link